src/go/k8s/api/vectorized/v1alpha1/console_types.go

```diff
@@ -160,6 +160,11 @@ type Deployment struct {
 
 	// +kubebuilder:default=1
 	MaxSurge int32 `json:"maxSurge,omitempty"`
+
+	// +optional
+	// Compute Resources required by the container.
+	// More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+	Resources corev1.ResourceRequirements `json:"resources,omitempty" protobuf:"bytes,8,opt,name=resources"`
```

Comment:
Couple nits here:
- Annotations should follow the comment, otherwise godoc will display the annotations as the documentation.
- Comments should be in the form of `{{ .FieldName }} ...`
- Optional and `omitempty` fields  _must_ be pointer values otherwise they'll default to an empty JSON object `{}` which can cause a lot of confusion.
- I don't _think_ there's a need for protobuf annotations. Let's stay consistent with our other fields.

```go
// Resources are the [corev1.ResourceRequirements] that are passed, verbatim, to the console Deployment.
// See: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
// +optional
Resources *corev1.ResourceRequirements `json:"resources,omitempty"`
```